CONTRIBUTING.md

```diff
@@ -2,52 +2,75 @@
 
 ## Code of Conduct
 
-All interactions with this project follow our [Code of Conduct][code-of-conduct].
-By participating, you are expected to honor this code. Violators can be banned
-from further participation in this project, or potentially all Coinbase projects.
+All interactions with this project follow our [Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code. Violators can be banned from further participation in this project, or potentially all Coinbase projects.
 
 [code-of-conduct]: https://github.com/coinbase/code-of-conduct
 
 ## Bug Reports
 
-* Ensure your issue [has not already been reported][1]. It may already be fixed!
-* Include the steps you carried out to produce the problem.
-* Include the behavior you observed along with the behavior you expected, and
-  why you expected it.
-* Include any relevant stack traces or debugging output.
+Before filing a bug report, ensure that your issue [has not already been reported][1]. It may already be fixed!
+
+If your bug hasn’t been fixed, follow these steps to file a bug report:
+
+1. If your issue is not reported or fixed, click the [*New*][5] button from the [*Issues*][1] page. A new interface will appear.
+2. Find _Bug report_ on the list, and click [_Get Started_][6]. A new interface will appear.
+3. Add a title for your bug report. It should briefly describe the problem.
+4. Follow the template that appears in the Write text box. This is the best way to describe the bug.
+* **Describe the bug** - A clear and concise description of what the bug is.
+* **To Reproduce** - Steps to reproduce the behavior.
+* **Expected behavior** - A clear and concise description of what you expected to happen.
+* **Additional context** - You can include any relevant stack traces or debugging output in this section.
+5. Click _Submit new issue_ to finish filing the bug report.
 
 ## Feature Requests
 
-We welcome feedback with or without pull requests. If you have an idea for how
-to improve the project, great! All we ask is that you take the time to write a
-clear and concise explanation of what need you are trying to solve. If you have
-thoughts on _how_ it can be solved, include those too!
+We welcome feedback with or without pull requests. If you have an idea for how to improve the project, great! All we ask is that you take the time to write a clear and concise explanation of the need you are trying to solve. If you have thoughts on _how_ it can be solved, include those too!
+
+To submit a feature request, follow these steps:
+
+1. Click the [*New*][5] button from the [*Issues*][1] page. A new interface will appear.
+2. Find _Feature request_ on the list, and click [_Get Started_][7]. A new interface will appear.
```

Comment:
Same `A new interface will appear` might not be needed.

Reply:
Updated in newest commit.